dashboard/src/shared/schema.ts

```diff
@@ -4,9 +4,9 @@
 import Ajv, { ErrorObject, JSONSchemaType } from "ajv";
 import * as jsonpatch from "fast-json-patch";
 import * as yaml from "js-yaml";
-import { isEmpty, set } from "lodash";
+import _ from "lodash";
```

Comment:
Why using `_` instead of importing the needed functions?

Reply:
You're right, we should import them explicitly (see [this benchmark](https://www.blazemeter.com/blog/import-lodash-libraries)). I just added some ongoing changes in my working branch, where I just used the usual `_`  import for lodash, but we should decrease the bundle size when possible. Thanks!